### ISSUE TITLE
handle spaces in token names

### ISSUE THREAD

**Issue by AlexandreBroudin** [opucuk/design-system-builder#3]
If I write 
```
typography: {
  base: {
    '1': {
      name: 'scale ratio',
      value: 1.125,
      unit: ''
    }
  }
}
```
the Sass output is : `$typography-base-scale ratio: 1.125;`
the expected output is : `$typography-base-scale-ratio: 1.125;`

sad dash is sad

**Comment by ugogo** [comment]
Nice catch!

Can you have a look at #4?
It should fix your error.

_Make the dash happy again_

**Comment by ugogo** [comment]
Fixed in #4
